src/controllers/organizations.js

```diff
@@ -195,8 +195,32 @@ async function updateOrgProfile(req, res) {
     }
 }
 
+async function deleteSubscription(req, res) {
+    try {
+        const { id: orgaId } = req.params;
+        const { id: userId } = req.user;
+        const user = await User.findById(userId);
+        if (user) {
+            if (Organization.findById(orgaId)) {
+                user.followedOrganizations.pull(orgaId);
+                user.save();
+            } else {
+                return res
+                    .status(404)
+                    .json({ error: 'Organization not found' });
+            }
+        } else {
+            return res.status(404).json({ error: 'User not found' });
+        }
+        return res.status(200).json({ message: 'Unsubscribed Successfully!' });
```

Comment:
In the other function, you used `Unfollowed Successfully!`. You can be consistent and also use `Unfollowed Successfully!` here.